packages/components/src/Workflow/subcomponents/Wrapper/Wrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropsWithChildren, HTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./Wrapper.module.scss"
+export type WorkflowWrapperProps = PropsWithChildren<
+  OverrideClassName<HTMLAttributes<HTMLDivElement>>
+>
+
```

Comment:
This update is less around the fix but a small DX improvement/continuity with the Main component we export that I thought we should provide